Fix description for Twitch auth (#201)
fastify/fastify-oauth2: README.md

}
```

## Provider Quirks

> This is a list of known quirks of some providers that needed additional work to get everything set up correctly.

Reviewer: ```suggestion
The following providers require additional work to be set up correctly.   
```

### Twitch

Twitch requires that the request for a token in the oauth2 flow contains the `client_id` and `client_secret` property.

> Unfortunately, this library cannot set this up automatically. Howevever, it's easy to configure:

Reviewer 2: Suggested change. (Sometimes it's good to point out what you're doing in both the text and the code example, as I know some devs read one or the other.)

"When authenticating via Twitch, include the `client_id` and `client_secret` in `tokenRequestParams`:"

Author: thanks, applied!

}
```

## Provider Quirks

The following providers require additional work to be set up correctly.   

### Twitch

Twitch requires that the request for a token in the oauth2 flow contains the `client_id` and `client_secret` property.

> Unfortunately, this library cannot set this up automatically. When authenticating via Twitch, include the

Reviewer 3: "Unfortunately"? Is there a possibility to set this up automatically? I dont think so. So I would remove that sentence with unforrtunately.

@Fdawgs You are a native speaker of traditional english. Wdyt?

Reviewer: Still valid, it is unfortunate it can't be setup automatically. 😊
